java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1152,6 +1153,94 @@ public void zscore(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void xadd(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String field1 = UUID.randomUUID().toString();
+        String field2 = UUID.randomUUID().toString();
+
+        assertNull(
+                client
+                        .xadd(
+                                key,
+                                Map.of(field1, "foo0", field2, "bar0"),
+                                StreamAddOptions.builder().makeStream(Boolean.FALSE).build())
+                        .get());
+
+        String timestamp1 = "0-1";
```

Comment:
nit
```suggestion
        String entryId = "0-1";
```

Reply:
timestamp1 is the variable name used in the node test :(